website/app/components/doc/icons-list/index.hbs

```diff
@@ -3,6 +3,10 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
+<p class="doc-text-body">Documentation on the Icon component is available
+  <LinkTo @route="show" @model="components/icon" class="doc-link-generic">here</LinkTo>.
```

Comment:
So for accessibility purposes, we don't want to say "link here" but rather try to say things like "see the documentation for the icon component" and make that the link text. Does that make sense?

Reply:
This does actually -- similar to what I did in resources.

Reply:
Yes. I hadn't considered that one before, but that makes sense. Thanks.